HED-schema-reduced.mediawiki

```diff
@@ -96,60 +95,69 @@ There are four ways to define the scope of an event.  The most common method is
 ** Siren
 ** Music
 ** Noise
-** Cocktail party <nowiki>[Multiple speech streams at once]</nowiki>
-** Environmental <nowiki>[A background sound from the surrounding environment. For example machine operating or people walking]</nowiki>
+** Crowd sound <nowiki>[Multiple speech streams at once]</nowiki>
+** Environmental sound<nowiki>[A background sound from the surrounding environment. For example machine operating or people walking]</nowiki>
 *** <nowiki># {takesValue} [A text-description of a complex environmental sound]</nowiki>
 
 '''Action'''<nowiki> {extensionAllowed} [May or may not be in response to a previous Sensory presentation event.]</nowiki> 
-* Motor
-** Intentional
-*** Breathe
-**** Inhale <nowiki>[A deliberate inbreath, e.g. in response to a task instruction. If incidental, use Action/Movement/Incidental/Inspiration]</nowiki>
-**** Exhale <nowiki>[A deliberate outbreath, e.g. in response to a task instruction. If incidental, use Action/Movement/Incidental/Expiration]</nowiki>
-**** Hold
-*** Head
-**** Nod
-*** Pedal
-**** Step
-***** Heel strike <nowiki>[Takes attribute Attribute/Spatial/Object side/Left or Attribute/Spatial/Object side/Right.]</nowiki>
-***** Toe off <nowiki>[Takes attribute Attribute/Spatial/Object side/Left or Attribute/Spatial/Object side/Right.]</nowiki>
-**** Run
-**** Walk
-**** Trot
-**** Jog
-**** Jump
-**** Kick
-**** Tab
-**** Press
-**** Dance
-*** Manual
-**** Drop <nowiki>[For example the participant drops a handheld ball]</nowiki>
-**** Press <nowiki>[As in pressing a button. Release may be quick or delayed. This can be used also for other types of pushing actions and can be grouped with Item/Object/Human/Body part tag to indicate which part of the body was used to press the button.]</nowiki> 
-**** Release <nowiki>[As in releasing a pressed button. Release events are not frequently recorded as Participant action events unless their timing is task-relevant.]</nowiki>
-**** Retract
-**** Tap <nowiki>[A quick press, followed by a quick release]</nowiki>
-**** Reach <nowiki>[Requires a spatial goal such as reaching to touch a button or to grasp something. A body stretch is not a reach.]</nowiki> 
-**** Touch <nowiki>[May follow a Reach event]</nowiki> 
-**** Grasp <nowiki>[Grasp an object in one or both hands]</nowiki>
-**** Point <nowiki>[Should be grouped with Item/Agent/Human/Body part/Hand/Finger tag to specify which finger was used for the action.]</nowiki> 
-*** Operate
-**** Steer
-**** Accelerate
-**** Brake
-**** Stop
-**** Engage
-***** Turn signal
-***** Headlights
-***** Horn
-***** Radio
-**** Teleoperate <nowiki>[Control an object that you are not aboard]</nowiki> 
-** Ocular
+* Move
+** Breathe
```

Comment:
Should we put "Move chest" instead? And "Breathe" as a child of it?

Reply:
I don't think so.  Breathe is the process of respiration.  I am taking it out of Locomote and put by itself.